Mathlib/Analysis/Normed/Ring/Lemmas.lean

```diff
@@ -214,3 +215,59 @@ instance Int.instNormOneClass : NormOneClass ℤ :=
 
 instance Int.instNormMulClass : NormMulClass ℤ :=
   ⟨fun a b ↦ by simp [← Int.norm_cast_real, abs_mul]⟩
+
+section NonUnital
+variable [NonUnitalNormedRing α] [NormMulClass α] {a : α}
+
+lemma antilipschitzWith_mul_left {a : α} (ha : a ≠ 0) : AntilipschitzWith (‖a‖₊⁻¹) (a * ·) :=
+  AntilipschitzWith.of_le_mul_dist fun _ _ ↦ by simp [dist_eq_norm, ← _root_.mul_sub, ha]
```

Comment:
Why are these `_root_`s needed here?

Reply:
... They aren't, it was just code copied verbatim from elsewhere.